src/core_plugins/kibana/public/discover/directives/unsupported_rollup.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment } from 'react';
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+
+export const DiscoverUnsupportedRollup = () => (
+  <Fragment>
+    <EuiSpacer size="xl" />
+
+    <EuiFlexGroup justifyContent="center">
+      <EuiFlexItem grow={false} className="discoverNoResults">
+        <EuiCallOut
+          title="OSS Kibana doesn't support rollup index patterns. Please use X-Pack to search this index pattern."
```

Comment:
Do you think users will understand "OSS Kibana"? I wonder if we should rephrase this to instead say something like: "Index Patterns based on rollup indices require the rollup plugin from X-Pack which is either not installed or disabled."

Reply:
I like this a lot. @gchaps thoughts?